packages/automated-dispute/src/services/eboActor.ts

```diff
@@ -421,13 +421,28 @@ export class EboActor {
         try {
             this.logger.info(`Settling dispute ${dispute.id}...`);
 
-            // OPTIMIZE: check for pledges to potentially save the ShouldBeEscalated error
+            const escalationData = await this.protocolProvider.read.getEscalation(request.id);
```

Comment:
We should wrap this `protocolProvider` call in a different try-catch, log the error and return as we don't want to compromise the whole actor because a `read` operation was not successful.
```suggestion
    try {
        const escalationData = await this.protocolProvider.read.getEscalation(request.id);
    } catch (err) {
        ....
    }
    
    try {
        // Logic to settle/escalate
    }
```